src/Microsoft.AspNetCore.SignalR.Common/Protocol/HandshakeProtocol.cs

```diff
@@ -17,26 +20,39 @@ public static class HandshakeProtocol
     {
         private const string ProtocolPropertyName = "protocol";
         private const string ProtocolVersionPropertyName = "version";
+        private const string MinorVersionPropertyName = "minorVersion";
         private const string ErrorPropertyName = "error";
         private const string TypePropertyName = "type";
 
         /// <summary>
-        /// The serialized representation of a success handshake.
+        /// Previously used to cache the errorless return message.
+        /// Now, return messages need a minor version number, so multiple are cached in a dictionary.
```

Comment:
We generally don't want to talk about previous versions or future versions in docs as that quickly becomes out of date. How about:

The serialized representation of a successful handshake with no minor version value.

It's descriptive and the obsolete attribute does a better job of telling you not to use it :)